src/routes/anime/+layout.svelte

```diff
@@ -185,12 +185,13 @@
                                 arrow: false,
                                 allowHTML: true,
                                 placement: "bottom-end",
-                                offset: [0, 10],
                                 animation: "shift-away",
                                 appendTo: document.body,
                                 interactive: true,
                                 theme: "elaine",
                                 onTrigger: async (instance) => {
+                                    // set "1 vw"
+                                    instance.props.offset = [0, (1 * window.innerWidth) / 100]
```

Comment:
This should be calculated better. Add a `vw` fucntion

Reply:
Yea, are you sure? this is basic and maybe we dont use this more often

Reply:
Yea cause we might need this later